src/main/java/seedu/address/model/person/Person.java

```diff
@@ -102,35 +123,57 @@ public boolean equals(Object other) {
         }
 
         Person otherPerson = (Person) other;
-        return otherPerson.getName().equals(getName())
-                && otherPerson.getCurrentPlan().equals(getCurrentPlan())
-                && otherPerson.getLastMet().equals(getLastMet())
+      
+        return otherPerson.getClientId().equals(getClientId())
+                && otherPerson.getName().equals(getName())
                 && otherPerson.getPhone().equals(getPhone())
                 && otherPerson.getEmail().equals(getEmail())
                 && otherPerson.getAddress().equals(getAddress())
+                && otherPerson.getRiskAppetite().equals(getRiskAppetite())
+                && otherPerson.getDisposableIncome().equals(getDisposableIncome())
+                && otherPerson.getCurrentPlan().equals(getCurrentPlan())
+                && otherPerson.getLastMet().equals(getLastMet())
                 && otherPerson.getTags().equals(getTags());
+
     }
 
     @Override
     public int hashCode() {
         // use this method for custom fields hashing instead of implementing your own
-        return Objects.hash(name, phone, email, address, lastMet, currentPlan, tags);
+
+        return Objects.hash(name, phone, email, address, riskAppetite, disposableIncome, currentPlan, lastMet, tags);
     }
 
     @Override
     public String toString() {
         final StringBuilder builder = new StringBuilder();
-        builder.append(getName())
+
+        builder.append("Client ID: ")
+                .append(getClientId())
+                .append("; Name: ")
+                .append(getName())
+                .append("; Email: ")
+                .append(getEmail());
```

Comment:
Perhaps there should be no semicolon here?

```suggestion
                .append(getEmail())
```